sources/packages/web/src/views/institution/locations/confirmation-of-enrollment/LocationCOESummary.vue

```diff
@@ -41,9 +41,24 @@
         <c-o-e-summary-data
           :locationId="locationId"
           :enrollmentPeriod="EnrollmentPeriod.Upcoming"
-          header="Upcoming enrolment"
-          subTitle="These applications are still outside of the 21 days of the study start date."
-        />
+          header="Upcoming and
+        Previous Enrolments"
+        >
+          <template #coeSummarySubtitle>
+            <span>This page contains two types of records:</span>
+            <ul>
+              <li>
+                disbursements requests which are upcoming and will be required
```

Comment:
I can see you have used the same words given in your ticket. Are you sure that D in the Disbursements and the P in the previously need not be capitalized?. If you have already confirmed with @HRAGANBC, please ignore this comment. Thanks

Reply:
@guru-aot Checked with business (@CarlyCotton) and capitalized them.